tools/pytest.sh

```diff
@@ -30,28 +30,28 @@ fi
 
 export PYTEST_DISABLE_PLUGIN_AUTOLOAD="True"
 
-NO_XDIST="${NO_XDIST:-False}"
+RUN_XDIST="${RUN_XDIST:-False}"
 if [[ "${RDBMS:-}" == "sqlite" ]]; then
   # no parallel tests on sqlite, because of random "sqlite3.OperationalError: database is locked"
   echo "Disabling parallel testing for sqlite"
-  NO_XDIST="True"
+  RUN_XDIST="True"
```

Comment:
You should also flip the bool logic: if `NO_XDIST` is true, we don't run XDIST. But if `RUN_XDIST` is true, we run XDIST. These semantically have opposite meaning, so every instance should be flipped accordingly: e.g. `NO_XDIST` was `False` by default, so `RUN_XDIST` should be `True` by default, and so on.